lib/rouge.rb

```diff
@@ -4,104 +4,109 @@
 # stdlib
 require 'pathname'
 
-# The containing module for Rouge
-module Rouge
-  # cache value in a constant since `__dir__` allocates a new string
-  # on every call.
-  LIB_DIR = __dir__.freeze
+# Avoid multiple loads (top-level return will be possible when 2.4+ is the
+# minimum supported version)
+unless Object::const_defined?(:Rouge) && Rouge.const_defined?(:LIB_DIR)
```

Comment:
@jneen According to [these notes](https://rubyreferences.github.io/rubychanges/2.4.html#toplevel-return), this was the alternative approach before top-level return. What do you think?

Reply:
@jneen How does this look to you?